sliver/hostuuid/uuid_linux.go

```diff
@@ -0,0 +1,37 @@
+// +build linux
+
+package hostuuid
+
+/*
+  Sliver Implant Framework
+  Copyright (C) 2019  Bishop Fox
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import (
+	"fmt"
+	"io/ioutil"
+)
+
+func GetUUID() string {
+	uuid, err := ioutil.ReadFile("/var/lib/dbus/machine-id")
```

Comment:
I think it'd be better to switch to `/etc/machine-id`, as dbus could be missing on some systems.

Reply:
Should I keep dbus as a fallback?

Reply:
Can't hurt I guess. I just checked and the value is the same on Fedora.